operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -329,36 +336,43 @@ func mergeIOPSWithProfile(iop *iopv1alpha1.IstioOperator) (*v1alpha1.IstioOperat
 	if hub != "" && hub != "unknown" && tag != "" && tag != "unknown" {
 		buildHubTagOverlayYAML, err := helm.GenerateHubTagOverlay(hub, tag)
 		if err != nil {
+			metrics.CountCRMergeFail(metrics.CannotMergeProfileWithHubTagError)
 			return nil, err
 		}
 		profileYAML, err = util.OverlayYAML(profileYAML, buildHubTagOverlayYAML)
 		if err != nil {
+			metrics.CountCRMergeFail(metrics.CannotMergeProfileWithHubTagError)
 			return nil, err
 		}
 	}
 
 	overlayYAML, err := util.MarshalWithJSONPB(iop)
 	if err != nil {
+		metrics.CountCRMergeFail(metrics.CannotMarshalUserIOPError)
```

Comment:
same here. this is probably too low level to be meaningful to a user. IOPFormatError?

Reply:
Fixed.